docs/tutorials/cargo_tests/buffer/Makefile

```diff
@@ -0,0 +1,25 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+export CARGO_TARGET_DIR ?= target
+
+all: buffer_data.postcard
+
+bin/icu4x-datagen:
+	echo "Target dir: $(CARGO_TARGET_DIR)"
+	cargo install icu_datagen --root .
+
+bin/demo_buffer: Cargo.toml src/*.rs
```

Comment:
This doesn't capture dependency changes. Use `FORCE` to let cargo figure out if it needs to rebuild.

You should then remove the `cp` below and have the postcard rule use the binary from the target dir directly. Otherwise the mtime will change too much.

Reply:
I can use `FORCE` and then `cp -p` to not touch the mtime when copying. I still need the `cp` because I can't point the next task to depend on a file in an arbitrary directory (CARGO_TARGET_DIR).

Reply:
Also I'm removing the `cargo build --release` from the Makefile because the actual invocation of that command could vary (such as the cargo config file)

Reply:
Hmm, that's a downside of passing the config file in the command. You could put it back in `.cargo/config.toml` and in the crates.io CI rename it to `config.toml.disabled` to disable it.

Reply:
Strongly prefer a solution that doesn't involve changing the filesystem.

Reply:
can you disable it on the command line? `--config patch = []` or `--config.patch=[]`?